templates/new_structure/Pipeline.yml

```diff
@@ -0,0 +1,40 @@
+pool:
+    vmImage: ubuntu-latest
+
+trigger: none
+#  - main
+#  - auto/*
+    
+resources:
+  - repo: self
+
+stages:
+  - template: BuildStage.yml
+    parameters:
+      workingDirectory: '$(System.DefaultWorkingDirectory)/terraform'
+      checkovImageName: 'bridgecrew/checkov'
+      checkovTestResultsFileName: 'CheckovReport.xml' 
+      scanDirectory: 'terraform'
+      terrascanImageName: 'accurics/terrascan'
+      tfsecImageName: 'aquasec/tfsec'
+      skipCheck: none
+      dependsOn: 'runCheckov'
+      environment: 'dev'
+#        privatePoolName: 'cspmigtest2-build-agents'
```

Comment:
Same comment as before, but we're going to need to support this.

Reply:
Fixed in new commits.